addons/im_livechat/static/tests/livechat_channel_info_list.test.js

```diff
@@ -0,0 +1,62 @@
+import { contains, openDiscuss, start, startServer } from "@mail/../tests/mail_test_helpers";
+import { describe, test } from "@odoo/hoot";
+import { Command, serverState } from "@web/../tests/web_test_helpers";
+import { defineLivechatModels } from "@im_livechat/../tests/livechat_test_helpers";
+
+describe.current.tags("desktop");
+defineLivechatModels();
+
+test("display channel info list in Discuss by default with channel type `livechat`", async () => {
+    const pyEnv = await startServer();
+    const userId = pyEnv["res.users"].create({ name: "James" });
+    pyEnv["res.partner"].create({
+        name: "James",
+        user_ids: [userId],
+    });
+    const countryId = pyEnv["res.country"].create({ code: "be", name: "Belgium" });
+    const guestId = pyEnv["mail.guest"].create({
+        name: "Visitor #20",
+    });
+    const channelId = pyEnv["discuss.channel"].create({
+        anonymous_name: "Visitor #20",
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ guest_id: guestId }),
+        ],
+        country_id: countryId,
+        channel_type: "livechat",
+        livechat_operator_id: serverState.partnerId,
+        livechat_active: true,
+    });
+    await start();
+    await openDiscuss(channelId);
+    await contains(".o-livechat-ChannelInfoList");
+});
```

Comment:
kind of useless, I mean I get the idea of having close to "unit" test, but it's a lot of setup for something very small, which is already implicitly covered by the other test too